RFC/Deprecated/readme.md

```diff
@@ -0,0 +1,5 @@
+## Deprecated
```

Comment:
could you update each file to have 120/100 char limit? This improves the diff in future and it will be easier to read too.